ext/opcache/Optimizer/sccp.c

```diff
@@ -1004,7 +1018,8 @@ static inline int ct_eval_func_call(
 				}
 			}
 			/* pass */
-		} else if (zend_string_equals_literal(name, "version_compare")) {
+		} else if (zend_string_equals_literal(name, "version_compare")
```

Comment:
Out of the scope of this PR: **This looks like it will need to check if the version comparer is an allowed string.** In PHP 7, this was fine, but in PHP 8, it throws a ValueError, which should be done at compile time, not runtime

```
php > var_export(version_compare('a', 'b', '>==='));
Warning: Uncaught ValueError: version_compare(): Argument #3 ($operator) must be a valid comparison operator in php shell code:1
Stack trace:
#0 php shell code(1): version_compare('a', 'b', '>===')
#1 {main}
  thrown in php shell code on line 1
```